src/vs/editor/contrib/parameterHints/parameterHintsWidget.ts

```diff
@@ -237,7 +241,11 @@ export class ParameterHintsWidget extends Disposable implements IContentWidget {
 		} else if (typeof signature.documentation === 'string') {
 			dom.append(this.domNodes.docs, $('p', {}, signature.documentation));
 		} else {
-			const renderedContents = this.renderDisposeables.add(this.markdownRenderer.render(signature.documentation));
+			const renderedContents = this.renderDisposeables.add(this.markdownRenderer.render(signature.documentation, {
+				asyncRenderCallback: () => {
+					this.domNodes?.scrollbar.scanDomNode();
+				}
+			}));
```

Comment:
This line and the lines surrounding lines are a duplicate of line 228-232.  Other than the dom element target (documentation or dom variable).  Should we refactor to bring these into the same place e.g.

```
		if (typeof signature.documentation === 'string') {
			dom.append(this.domNodes.docs, $('p', {}, signature.documentation));
		} else {
			const renderedContents = this.renderDisposeables.add(this.markdownRenderer.render(signature.documentation));
			const renderedContents = this.renderDisposeables.add(this.markdownRenderer.render(signature.documentation, {
				asyncRenderCallback: () => {
					this.domNodes?.scrollbar.scanDomNode();
				}
			}));
			renderedContents.element.classList.add('markdown-docs');
			const target = activeParameter?.documentation ?? dom;
			target.append(this.domNodes.docs, renderedContents.element);
```

Reply:
just like this?
```typescript
private renderMarkdownDocs(markdown: IMarkdownString | undefined): IMarkdownRenderResult {
		const renderedContents = this.renderDisposeables.add(this.markdownRenderer.render(markdown, {
			asyncRenderCallback: () => {
				this.domNodes?.scrollbar.scanDomNode();
			}
		}));
		renderedContents.element.classList.add('markdown-docs');
		return renderedContents;
	}
```